lib/manager/npm/post-update/yarn.ts

```diff
@@ -1,13 +1,13 @@
 import is from '@sindresorhus/is';
-import { validRange } from 'semver';
+import { readFile, remove } from 'fs-extra';
```

Comment:
Why this change? Please use `lib/util/fs` instead.

Reply:
sorry, probably a wrong merge